toolkits/zoom/pyproject.toml

```diff
@@ -1,30 +1,37 @@
-[tool.poetry]
+[build-system]
+requires = [ "hatchling",]
+build-backend = "hatchling.build"
+
+[project]
 name = "arcade_zoom"
 version = "0.1.10"
 description = "Arcade.dev LLM tools for Zoom"
-authors = ["Arcade <dev@arcade.dev>"]
+requires-python = ">=3.10"
+dependencies = [ "arcade-tdk>=0.1.0,<1.0.0", "httpx>=0.27.2,<1.0.0",]
+[[project.authors]]
+name = "Arcade"
+email = "dev@arcade.dev"
 
-[tool.poetry.dependencies]
-python = "^3.10"
-arcade-ai = ">=0.1,<2.0"
-httpx = "^0.27.2"
+[project.optional-dependencies]
+dev = [
+    "arcade-ai[evals]",
```

Comment:
TODO for all toolkits. Once we know the version of arcade-ai that we will be releasing, then restrict to that major version